frontend/src/exams/add-exam-form-components.js

```diff
@@ -182,8 +188,8 @@ export const ExamReceivedQuestion = Vue.component('exam-received-question', {
             <span v-if="error" style="color: red">{{ validationObj[q.key].message }}</span>
           </label><br>
           <b-form-radio-group v-model="showRadio"
-                              @input="preSetDate"
-                              :options="addExamModal.setup === 'other' ? otherOptions : options" />
+                              @input="preSetDate()"
```

Comment:
I didn't think you passed the actual function here, with `()` at the end?

Reply:
yeah that doesn't really matter here.   if you give it a function reference it will be able to add any/all applicable arguments, if you give it a function call it will call the function with only the arguments you supply or none and there are none here.